CanHazFunny/CanHazFunny/IJokeOutput.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace CanHazFunny
+{
+	public interface IJokeOutput
+	{
+		public void outputJoke(string joke);
```

Comment:
Avoid specifying access modifiers on interfaces. Methods should be PascalCase
```suggestion
		public void OutputJoke(string joke);
```